syncer/syncer.go

```diff
@@ -979,6 +1006,11 @@ func (s *Syncer) addJob(job *job) error {
 		startTime := time.Now()
 		s.tctx.L().Debug("queue for key", zap.Int("queue", queueBucket), zap.String("key", job.key))
 		s.jobs[queueBucket] <- job
+		s.isTransactionEnd.Store(false)
```

Comment:
should also set transaction end to true after ddl event

Reply:
> from lance https://github.com/pingcap/dm/pull/1928#discussion_r684867458
> we initialized `s.isTransactionEnd.Store(true)`, and there should always be a XID event after DML events commit, so I think this logic to handle DDL event is not needed.